File: lab-8/audio.py
from scipy.io import wavfile
import numpy as np

input_file ="C:/Users/Hrushi'/Documents/Computational lab/lab-8/Chorus.wav"
output_file = "C:/Users/Hrushi'/Documents/Computational lab/lab-8/wave.wav"

fs, y = wavfile.read(input_file)
reversedata = np.flipud(y)

wavfile.write(output_file, fs, reversedata)

File: lab-8/audio.m
input_file = "C:/Users/Hrushi'/Documents/Computational lab/lab-8/Chorus.wav"
output_file = "C:/Users/Hrushi'/Documents/Computational lab/lab-8/wave.wav"
[y,fs] = audioread(input_file);
reversedata = flipud(y);
audiowrite(output_file, reversedata, fs);
